peer/lifecycle/lifecycle.proto

```diff
@@ -188,6 +188,30 @@ message QueryApprovedChaincodeDefinitionResult {
     ChaincodeSource source = 8;
 }
 
+// QueryApprovedChaincodeDefinitionsArgs is the message used as arguments to
+// `_lifecycle.QueryApprovedChaincodeDefinitions`.
+message QueryApprovedChaincodeDefinitionsArgs {
+    string name = 1;
+    int64 sequence = 2;
```

Comment:
I'm confused, if the new function doesn't require client to pass in name and sequence, why are they listed as input parameters here?

Reply:
@denyeart Thank you for your review. I apologize for the oversight; I was aware of the need to remove these parameters but inadvertently missed updating them in the final submission. You are correct that these input parameters are not necessary for this function. I have now corrected this and resubmitted the patch. Please take a moment to review the updated version.